server/game/cards/08-MotC/TheSpearRushesForth.js

```diff
@@ -0,0 +1,25 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { CardTypes, Players } = require('../../Constants');
+
+class TheSpearRushesForth extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Bow a participating character',
+            condition: () => this.game.isDuringConflict(),
```

Comment:
Needs to be only during a military conflict

Reply:
I totally missed that, thanks.